src/pyopticon/_system/_data_logging_widget.py

```diff
@@ -69,6 +69,10 @@ def _toggle_logging(self):
             self.logging_interlock = True # This prevents one particular bug with slow polling rates
             self.root.after(self.delay*1000, self._flip_interlock) # Kludgy, sorry
             self.open_file.close()
+
+            # Notifies observers when logging is stopped - enables Baserow widget to respond to data logging
```

Comment:
Similar to above, the baserow widget is (currently) private to us so we don't want to include info about it in Pyopticon itself. We can just remove that part of the comment.